cmd/deployments.go

```diff
@@ -111,6 +111,7 @@ var (
 		Usage:    "Trigger a deployment by archive",
 		Flags: []cli.Flag{appFlag,
 			cli.BoolFlag{Name: "war, w", Usage: "Specify that you want to deploy a WAR file"},
+			cli.BoolFlag{Name: "no-follow", Usage: "Disable the display of deployment process if the first API call succeeded"},
```

Comment:
I don't think the help message should talk about "API call". Our customers do not know the underlying logic of the `deploy` command, they don't know if there is some API calls or some other magic. What bother them is that when they call this command, it hangs for a long time, then displays the deployment logs, and eventually returns. Whereas there are some situations they want the command to return right after the deployment is triggered. What do you think about this wording:

```suggestion
			cli.BoolFlag{Name: "no-follow", Usage: "Return immediately after the deployment is triggered"},
```

Reply:
I would also update the `Description` of this command to add an example with this flag. Something like:

```
$ scalingo --app my-app deploy --no-follow archive.tar.gz v1.0.0
$ scalingo --app my-app deployment-follow
```
The `deployment-follow` call is what you should call if you want to follow the deployment you just triggered.

There is a "See also commands" message in the description and I would add `'deployment-follow'` in it.